frontend/packages/core/src/NPS/header.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import {
+  ClickAwayListener,
+  Grow as MuiGrow,
+  MenuList,
+  Paper as MuiPaper,
+  Popper as MuiPopper,
+} from "@material-ui/core";
+import ChatBubbleOutlineIcon from "@material-ui/icons/ChatBubbleOutline";
+import { sortBy } from "lodash";
+
+import type { Workflow } from "../AppProvider/workflow";
+import { IconButton } from "../button";
+import { useAppContext } from "../Contexts";
+import type { SelectOption } from "../Input";
+import styled from "../styled";
+
+import NPSFeedback from "./feedback";
+
+const Grow = styled(MuiGrow)((props: { placement: string }) => ({
+  transformOrigin: props.placement,
+}));
+
+const Popper = styled(MuiPopper)({
+  padding: "0 12px",
+  marginLeft: "12px",
+  zIndex: 1201,
+});
+
+const Paper = styled(MuiPaper)({
+  width: "420px",
```

Comment:
i know this is the figma spec but I think at this width it gives the appearance of too much white space. was playing around with diff widths and I think `350px` looks better and it also doesn't break existing styling (screenshot below). lmk what you think

<img width="479" alt="Screen Shot 2022-01-24 at 9 24 40 AM" src="https://user-images.githubusercontent.com/39421794/150800790-e0f4a03b-cdda-480a-954d-4787af18e0f5.png">

Reply:
Adjusted to 350